docker-compose.yml

```diff
@@ -9,4 +9,4 @@ services:
     volumes:
       - ./:/app
     working_dir: /app
-    command: "jekyll serve --host=0.0.0.0 --force_polling"
+    command: "jekyll serve --livereload --host=0.0.0.0 --force_polling"
```

Comment:
hope this doesn't create problems to the live environment, I've no idea :-D

Reply:
Uuuuh.. does live use docker?!

Reply:
I really don't know, probably not

Reply:
I thought this is just a 1:1 page directly to OpenMage.github.io as static pages - and the domain simply points to that github.io thingy. (Changes are live now aswell, so it was okay. And livereload is nice during developing. Sorry for changing too much out of scope of that ticked)

Reply:
Live definitely doesn't use Docker, I just added the docker file for local development.